### DISCUSSION TITLE
Not receiving "Forgot Password" email

### DISCUSSION THREAD

**Discussion by IQLofty90** [mastodon/mastodon#19775] [Q&A] [answered]
I forgot my password and I'm not receiving the reset email. It's not in my spam folder, and I'm certain I'm using the correct gmail address. Please help.

**Comment by Rhinogradentia** [comment, marked as answer]
I have the same problem (mastodon.online server), would be great if this could be solved.

Best

**Reply by aptaylor1** [reply]
I'm having the same problem. It's a wee bit frustrating.

**Reply by ElleEmDee** [reply]
Same. Good to know it's not just me

**Reply by Rhinogradentia** [reply]
I got it "solved" - it was pebkac - I used the wrong login server - mastodon.social instead of mastodon.online. Once I used the correct one I got the reset email for the password.

Best

**Reply by ElleEmDee** [reply]
THANK YOU. I hadn't logged in in years, so I had no idea that I was pinging a siloed server while trying to get a password reset. I was getting nothing on mastodon.social, but asking for a reset on mastodon.online did the trick.

**Comment by xerxn** [comment]
Same issue here, gmail address too. I've tried the password reset today and a few days ago. No email received, double checked the ol' spam folder too.

**Reply by xerxn** [reply]
I ended up signing out of the server I was on, hadn't done much anything yet anyhow, and then signed up on anew server. So, must have just been and issue with the server I was on.

**Reply by SailorKrypton** [reply]
Thanks!

**Comment by SailorKrypton** [comment]
I’m having the same problem. I recently signed up. I tried to edit my profile it crashed repeatedly. Uninstalled the app and then tried to sign in and I’ve requested a reset multiple a dozen times. Nada. Not in junk either.

**Comment by mohe2015** [comment]
You people should probably tell us on which server that was. Usually it is probably a server specific issue. Maybe its mailing system is not setup properly or its overloaded or they are blocked by your mail provider because of many emails from many new users or so

**Reply by SailorKrypton** [reply]
I tried on multiple servers but I think mainly Mastodon social and online. I finally received reset emails but when I tried to reset I received error messages telling me it had expired or once I was able to make a new password, when I entered it then I would get an error messsge and unable to to do it.

**Reply by xerxn** [reply]
I was on nerdculture.de

**Reply by mohe2015** [reply]
> I was on nerdculture.de

https://nerdculture.de/@order seems like the server had high load and updates the last few days. But they also have a support email linked in their about page so you could also ask there for help.

**Reply by wnewber** [reply]
The server I'm using - or rather failing to use since I can't log on until I can sort out the password issue - is mstdn.social.

**Reply by mohe2015** [reply]
> The server I'm using - or rather failing to use since I can't log on until I can sort out the password issue - is mstdn.social.

There is a mail address listed in the https://mstdn.social/about page so you could probably ask there for help. This should be the first general advice to people having issues I think (after having waited a few hours because some issues could be because of server load)

**Comment by SailorKrypton** [comment]
Awesome! Thanks so much!

**Comment by ghost** [comment]
newbie here... regarding the failure of receiving a verification email when i signed up... the weird part is i can 'login' but only through the one 'welcome' email i received - not a verification email, just a 'welcome' email... when attempting to sigh in, using the correct info, i keep getting the 'invalid email or password' error message... i click the 'forgot password' link, type in my email address, click submit but never receive a 'reset email'... does each server require different logins...? the email address and password i used to sign up is only viable on one server...? confusing and somewhat frustrating... also, from the login screen - why do i have to pick a server each time i login...?

**Reply by mohe2015** [reply]
Yes its only working on the server you signed up on. this is your home server. if your username is alice your account name would be `@alice@homeserver`. On every server there can be people with the username alice so there also could be `@alice@anotherhomeserver`.

**Reply by mohe2015** [reply]
Also which login page are you using? your home server should have its own login page.

**Comment by omwolfpack** [comment]
Doesn't look like this is being monitored/answered

**Reply by mohe2015** [reply]
There are literally several answers here. Most other questions dont bring anything new to the table so the general advice on the other answers should apply too. If you think there is something specific unanswered you can highlight that but please don't simply write that into the whole discussion here.

**Comment by omwolfpack** [comment]
Hi mohe2015... Thanks for responding and responding so quickly. Both are always appreciated. My bad, I jumped the gun on my comment as I did not read every comment. I saw folks suggesting and trying different things, couple that w/ the main thread still saying "Unanswered" and last comment like 14 days ago. I did go back and read every comment and sniff through every response and made sure I was hitting the forgot pw link from my server and it worked.

Thanks again
